cmd/pmm-managed-starlark/main.go

```diff
@@ -46,4 +58,59 @@ func main() {
 	if on, _ := strconv.ParseBool(os.Getenv("PMM_TRACE")); on {
 		logrus.SetLevel(logrus.TraceLevel)
 	}
+
+	l := logrus.WithField("component", "pmm-managed-starlark")
+
+	decoder := json.NewDecoder(os.Stdin)
+	var data checks.StarlarkScriptData
+	err := decoder.Decode(&data)
+	if err != nil {
+		l.Errorf("Error decoding json data: %s", err)
+		os.Exit(1)
+	}
+
+	err = runChecks(l, data)
```

Comment:
Let's make `runChecks` return results so both stdin and stdout are in the same function